e2e/playwright/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "plugin-e2e-tests",
+  "scripts": {
+    "test": "PW_SLOMO=200 npm run test --prefix ../../../mattermost-server/e2e/playwright -- --project=chrome --config='../../../mattermost-plugin-github/e2e/playwright/playwright.config.ts'",
```

Comment:
On my local machine, the mm server directory is in a different play. Maybe we could use `MM_SERVER_PATH` here, similar to how the apps plugin runs it's go e2e tests?